boards/nucleo32-f031/include/periph_conf.h

```diff
@@ -174,21 +174,6 @@ static const spi_conf_t spi_config[] = {
 #define RTC_NUMOF           (0U)
 /** @} */
 
-/**
- * @name   ADC configuration
- * @{
- */
-#define ADC_CONFIG {            \
```

Comment:
yes, but as I see it, the existing config was incomplete and this PR fixes that...